tests/security/mariadb/mariadb_ssl_server.pm

```diff
@@ -0,0 +1,55 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Setup mariadb server enviroment and wait client to connect
+# Maintainer: Starry Wang <starry.wang@suse.com> Ben Chou <bchou@suse.com>
+# Tags: poo#109154
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use Utils::Architectures;
+use lockapi;
+use mmapi 'wait_for_children';
+
+sub run {
+    my ($self) = @_;
+    select_console 'root-console';
```

Comment:
Please move "select_console 'root-console';" out of the "parameters definition area" and put it into bellow "code area".

Reply:
Done.